src/cli.rs

```diff
@@ -9,20 +9,24 @@
 
 mod hex_utils;
 use std::{
-    fs::File,
-    io::{self, Read},
+    io::{self}
 };
 
+use std::{fs};
+
+use serde::{Deserialize, Serialize};
+
 use clap::{App, Arg};
 use sensei::GetBalanceRequest;
 use sensei::{admin_client::AdminClient, node_client::NodeClient};
 use tonic::{metadata::MetadataValue, transport::Channel, Request};
 
+
 use crate::sensei::{
     CloseChannelRequest, ConnectPeerRequest, CreateAdminRequest, CreateInvoiceRequest,
     CreateNodeRequest, GetUnusedAddressRequest, InfoRequest, KeysendRequest, ListChannelsRequest,
     ListNodesRequest, ListPaymentsRequest, ListPeersRequest, OpenChannelRequest, PayInvoiceRequest,
-    SignMessageRequest, StartAdminRequest, StartNodeRequest,
+    SignMessageRequest, StartAdminRequest, StartNodeRequest, PaginationRequest
```

Comment:
This is needed (I think) in order to make the CLI actually be able to send paginated requests (which allows, the list of nodes to be more than 0 items long).